src/cljc/athens/common_events/bfs.cljc

```diff
@@ -128,6 +128,8 @@
   "Convert internal representation to the vector of atomic operations that would create it.
   :block/save operations are grouped at the end so that any ref'd entities are already created."
   [db internal-representation default-position]
+  (when-not (vector? internal-representation)
```

Comment:
This will cause some notification ops to fail. I saw that some of these were lists, not vectors. @sid597

Reply:
Fixed by not throwing on lists